blog/2024/05-05-jellyfin-release-10.9.0/index.mdx

```diff
@@ -0,0 +1,133 @@
+
+* Uploaded images are now subject to much more verification, helping plug several security holes in this feature.
+
+* The backend SQLite database now supports connection pooling, which should improve performance for database operations.
+
+* Support for the HappyEyeballs engine improving dual-stack IPv4/IPV6 network support.
+
+* Improved handling of IP binding to fully respect these settings (including DLNA and SSDP).
+
+### Packaging
+
+* The `jellyfin` service user is now automatically added to the `render` and `video` groups on Debian/Ubuntu systems for new installs, helping improve the onboarding for hardware acceleration.
+
+* Packaging is now unified with a new CI system, as outlined in [our previous blog post](https://jellyfin.org/posts/new-ci-new-repo). As a normal user you should see no change here, but this helps massively improve the build process, and if you are building your own packages please reference [the new packaging repository](https://github.com/jellyfin/jellyfin-packaging) for the updated process.
+
+### Transcoding & FFmpeg
+
+* The `--ffmpeg` command-line flag is now the primary method of setting FFmpeg paths, and configuration of the FFmpeg paths via `encoding.xml` is now deprecated. All official packaging will revert to defaults unless explicitly set before upgrade.
+
+* Several audio improvements: support for Opus, FLAC, and ALAC codecs in the HLS engine; improved support and handling for transcoding DTS and TrueHD audio streams; new codec enforcement; better bitrate calculations; new audio normalization features; and stereo downmixing algorithm selection.
+
+* FFmpeg segments can now be automatically deleted after the client requests them, significantly reducing the space requirements of the transcoding temporary directory. This optional feature is disabled by default and can be enabled in the transcoding settings.
+
+* Support for direct stream playback of DVD and BluRay data folders, preferred over ISOs.
+
+* Support for AV1 hardware and software encoding.
+
+* Support for RK3588 hardware acceleration via MPP.
+
+* Support for Vulkan-based tonemapping with AMD VA-API.
+
+* Support for complete hardware acceleration on Apple systems (both Apple Silicon and Intel; previously was partial).
+
+* Support for Dolby Vision playback remuxing.
+
+### Scanning, Library & Playlist Management
+
+* Support for scanning and displaying lyrics.
+
+* Season names are now parsed from NFO files.
+
+* The MusicBrainz interface has been improved to better handle music libraries.
+
+* Support for hearing-impaired subtitles (sdh/hi./cc).
+
+* Support for playlist ACLs with playlist sharing between users and enforcement of playlist ownership. Existing playlists will become owner-only, and playlist permissions can be set when made global/shared. Playlists from library files will be global.
```

Comment:
Note: not merged yet for web, if we don't get it in time we should remove it from the changelog (changelog should only mention stuff available in the reference client)

Reply:
Do we still think it'll land? If not we can just remove it.

Reply:
Bill says he is working on it, so it might still land. That's why I left it as a note.

Reply:
(context) We're 2/3 PRs in, the last one should land soon for all this to work so it will get in. We were basically forced to or we'd completely break playlists with the API change, so.

Reply:
We still won't have a UI for setting the user level permissions.

Reply:
```suggestion
* The visibility of playlists can now be made private
  * User playlists are now private by default
  * We're working on playlist sharing for a future release
```

Reply:
"playlist sharing" is fine I think... you just can't set permissions for individual users... and you can only set a playlist as public/private when it is created

so you can mark a new playlist as public and any user can view it but not edit it since that is a user level perm